lib/node_modules/@stdlib/math/base/special/max/benchmark/c/benchmark.c

```diff
@@ -134,4 +136,4 @@ int main( void ) {
 		printf( "ok %d benchmark finished\n", i+1 );
 	}
 	print_summary( REPEATS, REPEATS );
-}
+}
```

Comment:
we need no change in this file as well

Reply:
Resolved

Reply:
I don't think this is done yet, I can see the changes.